group_vars/all/all.yml

```diff
@@ -53,7 +53,7 @@ ckan_fqdn: "ckan.minikube"
 ckan_site_url: "http://{{ ckan_fqdn }}"
 ckan_image: "ghcr.io/fjelltopp/fjelltopp-base-images/ckan"
 ckan_image_tag: "2.9.8"
-ckan_db_image: "postgres"
+ckan_db_image: "postgres:15"
```

Comment:
Have we confirmed that we don't overwrite this anywhere else?

Reply:
It is the default place where db image is set. It can be overridden by group_vars but I don't think any project is doing it.

Reply:
:+1: just checking we didn't need a partner PR in fjelltoppp-infrastructure